### PR TITLE
Support none port option

### PR DESCRIPTION
**Pull request by raaaaaaaaaan**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -17,6 +17,9 @@ export default class BlobbyS3 {
       style: 'path',
       agent: undefined // use http.globalAgent by default
     }, opts);
+    if (this.options.port === 'none') {
```
Comment (asilvas) [inline src/index.js:20]: what is the value of this?

Comment (raaaaaaaaaan) [reply src/index.js:20]: In our CEPH config file, we don't have `port` filed, using default port (80) will cause error while getting photos.

You can check an example in `blobby-api/pull/12/files` file `config/development.json5`

Comment (asilvas) [reply src/index.js:20]: What is the port then? It's required one way or another, be it from defaults or otherwise.

Comment (raaaaaaaaaan) [reply src/index.js:20]: hmm, we don't have `port` value in our CEPH config, will ask CEPH team if they know it

Comment (raaaaaaaaaan) [reply src/index.js:20]: get the port from ceph team, will update blobby-api config files and close this PR

**Pull request closed without merging**